File: Strings/Easy/1119. Remove Vowels from a String.java
class Solution {
    public String removeVowels(String s) {
        
        String ans = "";
        
        for(char c: s.toCharArray()){
            if(c != 'a' && c != 'e' && c != 'i' && c != 'o' && c != 'u'){
                ans += c;
            }
        }

        return ans;
    }
}

File: Strings/Easy/C++/1119. Remove Vowels from a String.cpp
class Solution {
public:
    string removeVowels(string s) {
        string ans = "";

        for(int i = 0; i < s.length(); i++){
            char ch = s.at(i);
            if(ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u'){
                continue;
            }
            ans += s.at(i);
        }

        return ans;
    }
};
